apps/compound/src/abis/CompoundLens.ts

```diff
@@ -0,0 +1,373 @@
+const CompoundLensABI = [
```

Comment:
I haven't looked into it but probably we can remove some methods from the abi we don't use to reduce the bundle size

Reply:
I was adding full size abis. Yeah totally true. I left only the methods i need